src/main/java/io/candydoc/plugin/save_ubiquitous_language/file/yml/UbiquitousLanguageAsYml.java

```diff
@@ -1,12 +1,11 @@
 package io.candydoc.plugin.save_ubiquitous_language.file.yml;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
 import io.candydoc.plugin.save_ubiquitous_language.file.UbiquitousLanguageAsFile;
+import io.candydoc.plugin.save_ubiquitous_language.file.strategy.SerializationStrategyFactory;
 import java.nio.file.Path;
 
 public class UbiquitousLanguageAsYml extends UbiquitousLanguageAsFile {
   public UbiquitousLanguageAsYml(Path destinationFolder) {
-    super(new ObjectMapper(new YAMLFactory()), destinationFolder, "yml");
+    super(SerializationStrategyFactory.forYml(destinationFolder).initDirectory());
```

Comment:
As the `initDirectory` is set in both Yml and Json implementations, I think it could be done by the UbitiquousLanguageAsFile:

``` java
class UbiquitousLanguageAsFile {
  UbiquitousLanguageAsFile(SerializationStrategy strategy) {
    strategy.initDirectory();
  }
}
```